armi/nucDirectory/tests/test_nuclideBases.py

```diff
@@ -185,6 +191,12 @@ def test_nucBases_imposeBurnChainTransmutationBulkStatistics(self):
             )  # ternary fission
 
     def test_nucBases_imposeBurn_nuSF(self):
+        """Test the natural abundance values.
```

Comment:
It doesn't look like this test is related to the natural abundance values. Rather, it tests that the neutrons released per spontaneous fission.
https://github.com/terrapower/armi/blob/98566492147d6f5702233f89d308c279ff7e6353/armi/nucDirectory/nuclideBases.py#L294-L296

Reply:
Fair, I change the test crumb description.  The important bit is that this test probes data we got from this file:

https://github.com/terrapower/armi/blob/98566492147d6f5702233f89d308c279ff7e6353/armi/nucDirectory/nuclideBases.py#L1139

Thus proving that we stored the data separate from the code.

Reply:
Can you also update the docstring for the test since it's wrong?

Reply:
Done